lib/ReactViews/Map/Panels/panel.scss

```diff
@@ -41,6 +47,10 @@
     margin: 14px 0 0 0;
   }
 
+  @media (max-width: $sm) {
+    width: 384px !important;
+  }
+
```

Comment:
It would be better to avoid `!Important`. I'm not sure how to. It seems like a few panels have problem being wider than a mobile screen (but related maps has the most cut off)